### DISCUSSION TITLE
addons/middle-click-popup/

### DISCUSSION THREAD

**Discussion by giscus** [ScratchAddons/website-v2#464] [Page comments]
# addons/middle-click-popup/

https://scratchaddons.com/addons/middle-click-popup/

**Comment by iqnite** [comment]
https://scratchaddons.com/addons/middle-click-popup/#:~:text=Scratch%20Addon%20settings
This link is broken, it should be `#addon-middle-click-popup` instead of `#middle-click-popup`.

**Reply by WorldLanguages** [reply]
This was fixed for all wiki pages now :)

**Comment by FARIZ56** [comment]
hello the bar used to show up but now it is not working i am middle clicking

**Reply by FARIZ56** [reply]
it working again

**Comment by Asomeone42** [comment]
Whenever I try to pull up the menu my mouse gets stuck in the code area and doesn't pull up the insert menu

**Reply by roserohan75-star** [reply]
Me too

**Reply by Samq64** [reply]
This will be fixed in v1.45.1.
